chapter_linear-networks/linear-regression.md

```diff
@@ -247,18 +247,35 @@ import numpy as np
 import time
 ```
 
+```{.python .input}
+#@tab paddle
+%matplotlib inline
+from d2l import paddle as d2l
+import math
+import paddle
+import numpy as np
+import time
+```
+
 为了说明矢量化为什么如此重要，我们考虑(**对向量相加的两种方法**)。
 我们实例化两个全为1的10000维向量。
 在一种方法中，我们将使用Python的for循环遍历向量；
 在另一种方法中，我们将依赖对`+`的调用。
 
 ```{.python .input}
-#@tab all
+#@tab mxnet, pytorch, tensorflow
```

Comment:
is it possible to reuse this code? e.g., we can redefine d2l.ones() in paddle.py.